roles/zabbix_agent/tasks/Windows.yml

```diff
@@ -81,7 +81,7 @@
     - agent_file_info.stat.exists
 
 - name: "Windows | Uninstall Zabbix (Update)"
-  win_command: '{{ zabbix_win_exe_path }} --config {{ zabbix_win_install_dir }}\zabbix_agentd.conf --uninstall'
+  win_command: '"{{ zabbix_win_exe_path }}" -config "{{ zabbix_win_install_dir }}\zabbix_agentd.conf" --uninstall'
```

Comment:
sorry don't have windows but is change from `--config` to `-config` intended or typo? On L126 it is still `--config`

Reply:
@D3DeFi definetly typo, it should not change this part. Will check if I can correct that.

Reply:
Ok fixed that on the latest commit.